modal/parallel_map.py

```diff
@@ -527,25 +537,23 @@ async def drain_input_generator():
             async_map_ordered(input_iter(), create_input, concurrency=BLOB_MAX_PARALLELISM)
         ) as streamer:
             async for q_item in streamer:
+                # TODO(ben-okeefe): Handle retrying inputs.
```

Comment:
Could you explain why the input-plane version has this TODO, but the control-plane doesn't?